app/templates/brief.html

```diff
@@ -43,7 +43,41 @@
       with
       type = "temporary-message",
       heading = "This opportunity was withdrawn on " + "{}".format(brief.withdrawnAt|dateformat)|nbsp,
-      message = "You can't apply for this opportunity now. The buyer may publish an " + "updated version"|nbsp + " on the " + "Digital Marketplace"|nbsp
+      message = "You can't apply for this opportunity now. The buyer may publish an " + "updated version"|nbsp + " on the " + "Digital Marketplace."|nbsp
```

Comment:
I'm curious why `"updated version|nbsp"` is separated out here? Why not have it all in one string with `|nbsp` at the end (as per the heading on the line above). Also, why do we need the `|nbsp` on the message in the first place?  I can understand wanting the header all on one row but does the same apply for the message? Or is this required for the banner macro?

Reply:
That might be a question @benvand is better equipped to answer :)

Reply:
Ben did indeed answer my question 😸 Happy to keep this as it is.